### ISSUE TITLE
No word suggestions for keyboards

### ISSUE THREAD

**Issue by atcq** [nidhaloff/deep-translator-app#1]
On Android 10 (LOS 17.1 on OnePlus 6) i don't get the suggestion bar. Tried with SwiftKey and Android Keyboard (AOSP).

Read a little bit in the kivy documentation.
It seems like `keyboard_suggestions = True` should handle that for keyboards for the TextInput widget. In the Doc it seems it should default to true but is only available if `input_type = 'text'` which should also be as default. Though the version of kivy you're using is fairly old and the Doc was for the current stable version. Maybe you have to set it explicitly?

Nice little App though 👍🏻

**Comment by nidhaloff** [comment]
@atcq Thanks for the hint. I didn't notice this (I didn't even know that it is possible with kivy).
I'm already planning on improving the app, so I will definitely take your suggestion into account ;)

**Comment by atcq** [comment]
Sweet!
Trying to figure out github with pycharm right now and might play a little bit with your code if you don't mind. I need a little project to learn python besides codewars :)
Found a different issue as well, on switching between keyboards in the TextInput widget the backspace button stops working for the widget (switching translation service will fix it). If you like i can make a different issue tracker for that.

**Comment by nidhaloff** [comment]
@atcq Yes the project is open source. You can contribute to the project if you like. Feel free to fork the repo, work on the issue or new idea/enhancement and make a pull request. 
Yes you can make a separate issue tracker for that. Thanks

It would be cool to have a solid app, where users can switch between translators. Such an app does not exist as far as I know. That's why I made this app and open sourced it in the first place. So feel free anytime to help me improve it if you have time and will.
